exporters/jaeger/build.gradle

```diff
@@ -0,0 +1,63 @@
+description = 'OpenTelemetry - Jaeger Exporter'
+
+apply plugin: 'com.google.protobuf'
+
+def protobufVersion = '3.7.1'
+def protocVersion = '3.7.1'
+
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.8"
+    }
+}
+
+dependencies {
+    api project(':opentelemetry-api')
```

Comment:
Does it module depend on the API? I could not find any reference.

Reply:
You are right, apparently there's no dependency to the API :-)